index.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs')
 var path = require('path')
 
 function useYarn(message) {
-  if (isNpmNotYarn) {
+  if (isNpmNotYarn && process.env && process.env.DISABLE_USE_YARN !== 'true') {
```

Comment:
So if `process.env` were falsy (and I'm not sure if that's even possible in Node), then we would fail the check and error out? Seems like odd behavior.